src/components/small-card/small-card.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useState } from "react";
+import { ISmallCardProps } from "./small-card.interface";
+import { StylesWrapper } from "./small-card.styles";
+import { TransitionImage } from "../image";
+import { imagePlaceholder } from "@/shared/utils/image-placeholder";
+import { FavoriteButton } from "../favorite-button";
+import Link from "next/link";
+
+export const SmallCard: FC<ISmallCardProps> = ({
+  data,
+  session,
+  onLikeClick,
+}) => {
+  const [isLiked, setIsLiked] = useState<boolean>(false);
+
+  const handleGetLevel = (level: number) => {
```

Comment:
For handling this kind of key value situations, you can create an object in your constants file, that behaves like a dictionary for example in this case, we can do:

```
const levels = {
  1: 'Low',
  2: 'Medium',
  3: 'High'
}
```

Then in your function you can do:

```
const handleGetLevel = (level = 1) => {
  return levels[number];
```

You can also set a default, so the function always returns a value inside the dictionary, in case the level is undefined :)

Reply:
🥔 I've implemented this suggestion as well :)